packages/thumbprint-react/components/DatePicker/index.jsx

```diff
@@ -46,7 +47,7 @@ export default class DatePicker extends React.Component {
 
         if (allowMultiSelection) {
             if (selected) {
-                const selectedIndex = newSelectedDays.findIndex(selectedDay =>
+                const selectedIndex = findIndex(newSelectedDays, selectedDay =>
```

Comment:
Whoa, I wonder if this just never worked in IE 11? 🥴

This PR needs a CHANGELOG entry for Thumbprint React since it changes the source code.

Reply:
Yep, that would throw an error since that method is not available. The power of linting!